__tests__/api/replaceVariablesDuringStreaming.test.ts

```diff
@@ -0,0 +1,63 @@
+import { replaceVariablesDuringStreaming } from "../../pages/api/v1/answers";
+
+describe("replaceVariablesDuringStreaming", () => {
+  it("no variables", () => {
+    const out = replaceVariablesDuringStreaming(
+      "This is a test string",
+      "",
+      {},
+    );
+    expect(out.content).toEqual("This is a test string");
+    expect(out.valueVariableBuffer).toEqual("");
+  });
+  it("variable, but none in string", () => {
```

Comment:
can we have a test for replacement when the ID is fully in content? e.g. content=ID1

Reply:
Sure :smile: